Source/WebKit/GPUProcess/GPUProcess.cpp

```diff
@@ -249,6 +252,13 @@ void GPUProcess::initializeGPUProcess(GPUProcessCreationParameters&& parameters,
     SandboxExtension::consumePermanently(parameters.containerCachesDirectoryExtensionHandle);
     SandboxExtension::consumePermanently(parameters.containerTemporaryDirectoryExtensionHandle);
 #endif
+
+#if ENABLE(LLVM_PROFILE_GENERATION) && PLATFORM(IOS_FAMILY)
+    WebCore::initializeLLVMProfiling();
+    WebKit::initializeLLVMProfiling();
+    JSC::initializeLLVMProfiling();
```

Comment:
The order is strange here, as WebCore is between WebKit and JSC.

Reply:
Will fix that.